crates/turbopack-ecmascript/tests/analyzer/graph/pack-2521/graph-explained.snapshot

```diff
@@ -0,0 +1,27 @@
+err = ???*0*
+- *0* err
+  ⚠️  pattern without value
+
+errors = []
+
+path = (
+  | `../src/build/Release/sharp-${runtimePlatform}.node`
+  | "../src/build/Release/sharp-wasm32.node"
+  | `@img/sharp-${runtimePlatform}/sharp.node`
+  | "@img/sharp-wasm32/sharp.node"
+)
```

Comment:
It's unexpected that it already has this value here. It should have some kind of operation of `paths`. Maybe `path = Iterated(paths)`